PsevdoTwwiterApp/src/main/webapp/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="resources/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+    <title>PsevdoTwitter</title>
+
+   <script type="text/javascript" src="resources/js/script.js"></script>
+</head>
+<body>
+  
+  <button class="Log">Log out</button>
+  <h1>Valery</h1>
+ 
+   
+ <form action="/action_page.php">
+    <p class="filter">Filter</p>
+  <label for="Name">Name:</label><br>
+  <input type="text" id="fname" name="Name"  placeholder="Name" ><br>
+  <label for="Data">Date</label><br>
+  <input type="date" class="Data" id="Data" name="lname" placeholder="Data"><br>
+  <input type="date" class="Data2"  id="Data" name="lname"  placeholder="Data"><br>
```

Comment:
id is a unique attribute. You don't have to use the same id for several elements